app/Traits/UserTrait.php

```diff
@@ -53,6 +53,13 @@ public function partners()
         return $this->morphToMany(Partner::class, 'financeable');
     }
 
+    /** Get the application submission of the user. */
+    public function applicationSubmissions()
+    {
+        return $this->hasMany(ApplicationSubmission::class, 'dashboard_user_id')
+            ->orderBy('created_at', 'desc');
```

Comment:
I'd move the `orderBy` out of the relationship and into where we're actually using it.

```suggestion
```

This could also be turned into a scope on `App\Models\ApplicationSubmission`

```php
public function scopeLatest($query)
{
    return $query->orderBy('created_at', 'desc');
}
```